### ISSUE TITLE
Failed to install conan package from catchorg

### ISSUE THREAD

**Issue by chaplin89** [catchorg/Catch2#1505]
**Describe the bug**
If I try to install Catch2 from catchorg repository, the install fail with the following error:

> error : Requested 'Catch2/2.5.0@catchorg/stable' but found case incompatible 'catch2'

**Expected behavior**
I should be able to install Catch2

**Reproduction steps**

1.  Changed the dependency from: 
`catch2/2.5.0@bincrafters/stable`
To:
`Catch2/2.5.0@catchorg/stable`

1.  `conan remote add catchorg https://api.bintray.com/conan/catchorg/Catch2 True > nul 2>&1`

1.  `conan install . -g cmake_multi -s build_type=Release --build missing -if ./.dep`

**Platform information:**
Windows 10, MSVC 14, Catch 2.5.0

**Comment by horenmar** [comment]
#1501.

TLDR: conan does not deal well with case change in dependency, so you have to purge its cache by yourself.
